ansible/configs/kni-osp/files/clouds.yaml.j2

```diff
@@ -0,0 +1,12 @@
+clouds:
+  {{ osp_project_name }}:
+    auth:
+      auth_url: "{{ osp_auth_url }}"
+      username: "{{ osp_auth_username }}"
+      project_name: "{{ osp_project_name }}"
+      project_id: "{{ osp_project_id }}"
+      user_domain_name: "Default"
+      password: "{{ osp_auth_password }}"
```

Comment:
same...this should be something like `{{ heat_user_password }}` and not the admin password.

Reply:
this is how I found it in the tree I copied from, trying those changes

Reply:
leads to the following error when running the agnosticd config now : fatal: [provision]: FAILED! => {"changed": false, "msg": "AnsibleUndefinedVariable: 'heat_user_password' is undefined"}